test/e2e/suite/006-apps-in-any-namespace/01-install.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: image-updater-e2e-006
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: image-updater-e2e-006-01
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cmd-params-cm
+  namespace: argocd-image-updater-e2e
+  labels:
+    app.kubernetes.io/name: argocd-cmd-params-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  application.namespaces: image-updater-e2e-006-01
+---
+kind: AppProject
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: project-one
+  namespace: argocd-image-updater-e2e
+spec:
+  sourceNamespaces:
+  - image-updater-e2e-test
```

Comment:
looks like this ns should be `image-updater-e2e-006-01` to be in sync with the Application and cm.

Reply:
Yup.. fixed that in my latest commit.